packages/suite-native-settings/src/screens/SettingsScreen.tsx

```diff
@@ -1,37 +1,40 @@
 import React from 'react';
-import { Box, Button, Text } from '@suite-native/atoms';
-import { View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { SettingsStackRoutes, SettingsStackParamList } from '../navigation/routes';
 import { StackProps } from '@suite-native/navigation';
+import { VStack } from '@suite-native/atoms';
+
+import { ApplicationSettings } from '../components/ApplicationSettings';
+import { DeviceSettings } from '../components/DeviceSettings';
+import { CoinsSettings } from '../components/CoinsSettings';
+import { DeviceActionButtons } from '../components/DeviceActionButtons';
+import { SettingsStackParamList, SettingsStackRoutes } from '../navigation/routes';
 
-const settingsScreenStyle = prepareNativeStyle(() => ({
+const settingsSectionsStyle = prepareNativeStyle(utils => ({
     flex: 1,
     justifyContent: 'center',
-    alignItems: 'center',
+    padding: utils.spacings.medium,
 }));
 
 export const SettingsScreen = ({
     navigation,
 }: StackProps<SettingsStackParamList, SettingsStackRoutes.Settings>) => {
     const { applyStyle } = useNativeStyles();
 
+    const handleRedirect = (route: SettingsStackRoutes | undefined) => {
```

Comment:
redirect sounds weird do not have redirects :D pleease use `handleNavigate`

Reply:
[b7d1aa6](https://github.com/trezor/trezor-suite/pull/5617/commits/b7d1aa67768abc4a72c59c3d925329643a376396)